style-sheets/primevue-components-overrides.scss

```diff
@@ -686,29 +686,23 @@ ul#nav li.active a {
 }
 
 .#{variables.$primevue-prefix}-toast
-    .#{variables.$primevue-prefix}-toast-icon-close {
-    color: map.get(
-        lightNotifications.$light-notification-token-overrides,
-        'notification-action-tertiary-inverse-text'
-    ) !important;
-}
-
-.#{variables.$primevue-prefix}-toast
-    .#{variables.$primevue-prefix}-toast-icon-close
-    svg {
+    .#{variables.$primevue-prefix}-toast-icon-close svg{
     color: map.get(
         lightNotifications.$light-notification-token-overrides,
         'notification-action-tertiary-inverse-text'
     ) !important;
+    margin-top: 0 !important;
 }
 
 .#{variables.$primevue-prefix}-toast
     .#{variables.$primevue-prefix}-toast-icon-close:focus {
     box-shadow: 0 0 0 0.1rem map.get(lightTheme.$light-theme, 'support-error') !important;
 }
 
-.#{variables.$primevue-prefix}-toast-message-content .svg {
+.#{variables.$primevue-prefix}-toast-message-content svg {
     color: map.get(lightTheme.$light-theme, 'support-error');
```

Comment:
let's take out this line, and test the color in our own repo

Reply:
I removed it and did the local styling you suggested